controllers/controllers.js

```diff
@@ -1,13 +1,43 @@
-const { allCategories } = require("../models/models");
+const { allCategories, returnReview } = require("../models/models");
 
 exports.getCategories = (req, res, next) => {
   const { slug, description } = req.query;
   allCategories(slug, description)
     .then((categories) => {
-        console.log(categories)
       res.status(200).send({ categories });
     })
     .catch((err) => {
-        console.log(err, '<<< this is error')
+      console.log(err);
+    });
+};
+
+exports.getReview = (req, res, next) => {
+  const { review_id } = req.params;
+  const {
```

Comment:
If we are just querying the database by id, why is there a slew of arguments being passed to returnReview?? If I am looking for something in a table with an id of 1...do I need the other things?